addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2004,7 +2004,7 @@ msgstr ""
 
 #: addons/skin.estuary/xml/Variables.xml
 msgctxt "#427"
-msgid "Disc"
+msgid "Disc Number"
```

Comment:
Are you sure #427 isn't used somewhere else where  adding " number" is incorrect? Need to take great care with the old 3-digit ID messages